Code/Editor/ViewPane.cpp

```diff
@@ -465,7 +466,7 @@ void CLayoutViewPane::SetViewportFOV(const float fovDegrees)
         // if viewport camera is active, make selected fov new default
         if (pRenderViewport->GetViewManager()->GetCameraObjectId() == GUID_NULL)
         {
-            gSettings.viewports.fDefaultFov = fovRadians;
+            SandboxEditor::SetCameraDefaultFov(fovRadians);
```

Comment:
note: Here we use radians but it would be really easy to get confused and pass degrees instead, and arguable degrees is much friendly for humans to work with in the setreg file)